fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadJob.java

```diff
@@ -963,6 +968,13 @@ public TLoadInfo toThrift() {
             info.setNum_unselected_rows(loadingStatus.getLoadStatistic().totalUnselectedRows());
             info.setNum_scan_rows(loadingStatus.getLoadStatistic().totalSourceLoadRows());
             info.setNum_sink_rows(loadingStatus.getLoadStatistic().totalSinkLoadRows());
+            // warehouse
+            Warehouse warehouse = GlobalStateMgr.getCurrentState().getWarehouseMgr().getWarehouse(warehouseId);
+            if (warehouse != null) {
+                info.setWarehouse(warehouse.getName());
+            } else {
+                info.setWarehouse("");
```

Comment:
‌‌‌Within the implementation of the getWarehouse operation, checks are conducted. If a valid Warehouse is not obtained, an exception will be thrown directly. Therefore, we can omit the check here for whether it is null.
If we check the running mode, we should delete the else branch.